ui/src/containers/pipelines/ShowPipeline.js

```diff
@@ -74,6 +74,12 @@ class ShowPipeline extends Component {
             apiPath={`/pipelines/${pipeline.uuid}`}
             buttons={
               <>
+                <a
+                  href={`/pipelines/${pipeline.uuid}/edit-payload`}
+                  className="btn btn-primary text-white ms-2"
```

Comment:
I suggest using `btn-primary` for only one button. What about switching to `btn-secondary` here. What do you think?

Reply:
I had it as `btn-secondary` it was just confusing to me on why one of the buttons is colored differently as both are taking you to a way of editing the pipeline. But changed it :)